packages/core/src/store/index.ts

```diff
@@ -1,5 +1,7 @@
 import {
-  atom, /*AtomEffect,*/ RecoilRoot, useRecoilState,
+  atom,
+  /*AtomEffect,*/ RecoilRoot,
```

Comment:
```suggestion
  RecoilRoot,
```

前から残ってた消し忘れのやつなので、消していただけると 🙇